modules/sdk-coin-xlm/src/xlm.ts

```diff
@@ -708,15 +719,19 @@ export class Xlm extends BaseCoin {
     }
 
     const transaction: RecoveryTransaction = {
-      tx: Xlm.txToString(tx),
+      tx: Xlm.txToString(tx), // tx and txBase64 are the same, but OVC needs a field called txBase64
+      txBase64: Xlm.txToString(tx),
```

Comment:
This can be revisited to clean up so that we're not returning the two same fields.